src/components/PuzzleBoard/puzzleBoard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
This filename should be capitalized, since it returns a react component.

Reply:
So it should be 'import React from 'React'?

Reply:
Oh I see what you mean. Gotcha. I overlooked that.